scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/util/LongWrapper.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
```

Comment:
2019

Reply:
This is auto generated by gradle, i'll leave it to wait mael to update copyright of whole project.